### ISSUE TITLE
some or most recipes arent in the 0.9.3 for 1.19.2

### ISSUE THREAD

**Issue by TheGameSquad22** [DrMango14/Create-The_Factory_Must_Grow#123]
played in a custom hodgepodge modpack that was really slow, not the best, but i realised that the lightbulb recipe in JEI wasnt showing up, neither was the cable pipe. Neither was some other notable ones, like certain colors of the marked/hazard block you get from aluminum. unrelated note: 
Bauxite is really rare as i was searching for it too, kept finding either fireclay too much of the time or jasper from quark, or some really obscure one like Breccia from Dreams and Desires. (i have tectonic installed if that matters)

**Comment by WolfyWolftheAnimatronicWolf** [comment]
yeah, 99% of the recipes are gone on my end too... Found out when using JEI.

**Comment by DrMango14** [comment]
fixed
